osclib/stagingapi.py

```diff
@@ -508,6 +508,11 @@ def superseded_request(self, request, target_requests=None):
                     self.do_change_review_state(request_id, 'declined',
                                                 by_group=self.cstaging_group, message=message)
                 else:
+                    # Supersedes request is from the same project
+                    if request_new.find('./action/source').get('project') == request_old.find('./action/source').get('project'):
+                        message = 'sr#{} has newer source and is from the same project'.format(request_new.get('id'))
```

Comment:
I wouldn't lie - we just don't know if it's newer source. But it will do

Reply:
Yes, or we can re-use old message `Replaced by sr#XXX`